.changeset/dirty-moles-hammer.md

```diff
@@ -0,0 +1,5 @@
+---
+"@wso2is/console": patch
```

Comment:
Add i18n module as the change is in there.
Add myaccount as well since the it depends on i18n module

Reply:
fixed